Remove requirement for lsb

lsb drags in lots of useless package deps on Red Hat

diff -up scstadmin-2.2.1/Makefile.no-lsb scstadmin-2.2.1/Makefile
--- scstadmin-2.2.1/Makefile.no-lsb	2013-01-13 11:53:08.464087647 -0600
+++ scstadmin-2.2.1/Makefile	2014-07-29 13:36:58.542023974 -0500
@@ -61,11 +61,11 @@ install:
 	@if ! perl -MExtUtils::MakeMaker -e '' >/dev/null 2>&1; then \
 	echo Error: the Perl module ExtUtils::MakeMaker must be installed \
 	first; false; fi
-	@if [ ! -e /etc/slackware-version -a ! -e /etc/gentoo-release	      \
-	      -a ! -e /lib/lsb/init-functions ]; then			      \
-	echo "Error: the lsb-core (Debian, Ubuntu), redhat-lsb (RHEL, CentOS,"\
-	"Scientific Linux, Fedora) or insserv (SLES, openSUSE) package must"  \
-	"be installed first"; false; fi
+	# @if [ ! -e /etc/slackware-version -a ! -e /etc/gentoo-release	      \
+	#       -a ! -e /lib/lsb/init-functions ]; then			      \
+	# echo "Error: the lsb-core (Debian, Ubuntu), redhat-lsb (RHEL, CentOS,"\
+	# "Scientific Linux, Fedora) or insserv (SLES, openSUSE) package must"  \
+	# "be installed first"; false; fi
 	cd $(SCSTADMIN_DIR) && $(MAKE) $@
 	for m in scstadmin/man*/*.[1-9]; do				\
 	  if [ -e $$m ]; then						\
